docs/charts/bluecompute-ce/index.yaml

```diff
@@ -2,35 +2,35 @@ apiVersion: v1
 entries:
   bluecompute-ce:
   - apiVersion: v1
-    created: 2018-01-23T16:03:07.813067-06:00
+    created: 2018-02-14T11:00:10.397431-05:00
     description: Bluecompute Application Chart
-    digest: ad46bfcea93fa51dc6eac54bbc3a2c24e78a9f19c60dc9ca781baafb9ee9c2e5
+    digest: ad408ea385ae64492c05745e550a053765e3b51dfbfa4e629e17d220d3c56668
     name: bluecompute-ce
     urls:
-    - https://raw.githubusercontent.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes/master/docs/charts/bluecompute-ce/bluecompute-ce-0.0.5.tgz
+    - https://raw.githubusercontent.com/jkwong888/refarch-cloudnative-kubernetes/master/docs/charts/bluecompute-ce/bluecompute-ce-0.0.5.tgz
```

Comment:
@jkwong888 can you change back to `ibm-cloud-architecture` please? Other than that it looks good

Reply:
fixed